logbook-json/src/test/java/org/zalando/logbook/json/JsonPathBodyFiltersTest.java

```diff
@@ -28,13 +28,14 @@
 import static org.zalando.logbook.json.JsonBodyFilters.accessToken;
 import static org.zalando.logbook.json.JsonPathBodyFilters.jsonPath;
 
+@SuppressWarnings("UnstableApiUsage")
 class JsonPathBodyFiltersTest {
 
-    private final String type = "application/json";
```

Comment:
Those changes are not really necessary for the fix itself, are they?

Reply:
Yes, they are not necessary at all. I'll revert it